src/core/main.go

```diff
@@ -274,6 +278,7 @@ func main() {
 
 	log.Info("Fix empty subiss for meta info data.")
 	oidc.FixEmptySubIss(orm.Context())
+	scheduleSystemArtifactCleanJob(ctx)
```

Comment:
scheduleSystemArtifactCleanJob(ctx) might fail because the jobservice container might be not ready when harbor core is not ready.

Reply:
I proposed a possible solution in [here](https://github.com/goharbor/harbor/issues/17008)